xbmc/addons/AddonInstaller.h

```diff
@@ -26,40 +26,46 @@ namespace ADDON
 
 class CAddonDatabase;
 
-enum class BackgroundJob
+enum class BackgroundJob : bool
```

Comment:
If the enum will essentially be a bool might make more sense to have the enum values being TRUE and FALSE!

Reply:
that dont work because we have a name clash then with macros TRUE/FALSE.. was in the very beginning of our add-on system refactoring. yeah i do remember 🙂

Reply:
Everything comes full circle! Now you’re mentoring me!!!

Reply:
phunky, not really 😅 i know it was just a test that i passed now.. haha